ppx/test/input_ocaml.ml

```diff
@@ -17,3 +17,5 @@ let%component make ~children:(first, second) () =
   div ~children:[first; second] ()
 
 let%component make ?(name = "") = div ~children:[name] ()
+
+let%component make () = a ?href:(Some "https://opam.ocaml.org") ()
```

Comment:
Could we add a test in `test_jsoo_react` please? ppx tests will swallow all the things as long as they're syntactically valid, so they don't allow to detect type errors.

Reply:
I wanted to add them as example, but forgot we have tests that run the browser 💯 

Fixed in [`7c03b8b` (#92)](https://github.com/ml-in-barcelona/jsoo-react/pull/92/commits/7c03b8bf96a79f964dabf582fecf3e38c05f3b38)